requirements/main.in

```diff
@@ -30,6 +31,7 @@ pandas
 passlib[bcrypt]
 pymongo
 pydantic[email]>=1.10.0
+pytest-mock
```

Comment:
This is a test-only dependency right? It should only be in `dev.in` if that's the case.